vme/zone/spells.zon

```diff
@@ -8457,23 +8467,21 @@ code
       quit;
    }
 
-   hm := 1 + hm / 10;
-   if (hm > 10)
-      hm := 10;
+      hm := bonusCalc(hm);
 
-   addaff(tgt, ID_BLESS, 10, WAIT_SEC*30,
+   addaff(tgt, ID_BLESS, hm*2, WAIT_SEC*30,
```

Comment:
Why are you multiplying by two? Before it was hm/10. so now it's 2 * hm / 10. Is that intentional?

Reply:
used to be 10 all the time. There actually is a problem here, it should be hm, not hm*2 as duration is 1/2 of the rest of the boost spells. I'll edit that. Note that at this point hm is already modified by the boost calculation so it is the roll divided by 10 for the first 10 increments, then divided by an additional 3 (so effectively divided by 30) for everything beyond that. This can actually mean a shorter duration for really low rolls.. I'll actually min that too so that the minimum will still be 10.

Good catch!